std/asyncio/IReadable.hx

```diff
@@ -0,0 +1,21 @@
+package asyncio;
+
+import haxe.NoData;
+import haxe.io.Bytes;
+import haxe.Callback;
+
+/**
+	An interface to read bytes from a source of bytes.
+**/
```

Comment:
Perhaps something like a (strictly advisory) `prepare(size:Int)` method would make sense?

Reply:
What do you mean? `from Int to Int`?

Reply:
@RealyUniqueName I assume that comment was a reply to something else?

Reply:
> Perhaps something like a (strictly advisory) prepare(size:Int) method would make sense?

I don't know. Sounds like an implementation-specific.